packages/bpk-component-price/src/BpkPrice.js

```diff
@@ -147,7 +149,11 @@ const BpkPrice = (props: Props) => {
             !isAlignRight && 'bpk-price__spacing',
           )}
         >
-          <BpkText textStyle={priceTextStyle} tagName="span">
+          <BpkText
+            textStyle={priceTextStyle}
+            tagName="span"
+            data-price-your-pick={dataPriceYourPick ? 'true' : undefined}
```

Comment:
```suggestion
            {...dataAttributes}
```

The dataAttributes could be defined as

```
dataAttributes: Record<string, string>
```

when consume this component

```
<BpkPrice
        {...props}
        previousPrice={previousPrice}
        leadingText={leadingText}
        trailingText={trailingText}
       dataAttributes={ 'data-price-your-pick': 'true' }
      />
```

Has this property name been discussed and decided? If not, then I suggest that it should be more general, rather than designed for a specific need. For example, this actually adds [data attributes](https://developer.mozilla.org/en-US/docs/Learn_web_development/Howto/Solve_HTML_problems/Use_data_attributes) to this html element, so we can consider allowing custom data attributes, maybe they will be used in other scenarios in the future

Reply:
Yes, it would be better for general, I will update it.

Reply:
Have confirmed with backpack team we can make it.